docs/troubleshooting-development-issues.md

```diff
@@ -47,6 +47,10 @@ git clean -ifdX
    <img src="https://user-images.githubusercontent.com/1884376/94270515-ca579400-ff5d-11ea-8ff1-152cade31654.gif" alt="How to clean git untracked files">
 </details>
 
+### Issues with starting database
+
+If you are using a newer node version you may need to change the value of `MONGOHQ_URL` in your env file. Replace `localhost` with `0.0.0.0` and leave everything else the same
+
```

Comment:
Can you move this to `Issues with installing the recommended prerequisites` section, maybe line 14 🤔, and apply this suggestion? If you see it fitting.

```suggestion
#### Can't access the database with newer node versions

In your env file, change the value of `MONGOHQ_URL` to `mongodb://127.0.0.1:27017/freecodecamp`.

```